demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,136 @@
+# Python* Model API package
+
+Model API package is a set of wrapper classes for particular tasks and model architectures, simplifying data preprocess and postprocess as well as routine procedures (model loading, asynchronous execution, etc...)
+An application feeds model class with input data, then the model returns postprocessed output data in user-friendly format.
+
+## Package structure
+
+The Python* Model API consists of 3 libraries:
+* _adapters_ implements a common interface to allow Model API wrappers usage with different executors: OpenVINO, OVMS. See [Model API adapters](#model-api-adapters) section
```

Comment:
```suggestion
* _adapters_ implements a common interface to allow Model API wrappers usage with different executors. See [Model API adapters](#model-api-adapters) section
```
There shouldn't be examples. We will definitely forget to update here when more adapters will be introduced. Let's leave it only to the adapters section.